src/ipc-dbus/wpantund.conf

```diff
@@ -28,6 +28,10 @@
         <allow send_interface="com.nestlabs.WPANTunnelDriver.PermitJoin"/>
         <allow send_interface="com.nestlabs.WPANTunnelDriver.Status"/>
         <allow send_interface="com.nestlabs.WPANTunnelDriver.List"/>
+
+        <!-- Border Agent -->
+        <allow own="ot.ba"/>
```

Comment:
We should make this a real reverse DNS name. Not sure what reverse DNS name we should use, though. We are currently using `org.wpantund.*`, which would be appropriate if this is tied to wpantund.